tests/e2e_spin.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+import infra.ccf
+import infra.notification
+import suite.test_requirements as reqs
+import infra.logging_app as app
+import infra.e2e_args
+
+from loguru import logger as LOG
+
+
+@reqs.description("Check slow/expensive transactions do not cause an eleection")
+@reqs.supports_methods("SmallBank_spin")
+def test_large_messages(network, args):
+    primary, _ = network.find_primary()
+
+    with primary.node_client() as nc:
+        check_commit = infra.checker.Checker(nc)
+        check = infra.checker.Checker()
+
+        with primary.user_client(request_timeout=10) as c:
+            primary = c.rpc("getPrimaryInfo", {}).result["primary_host"]
+            c.rpc("SmallBank_spin", {"iterations": 800_000_000})
```

Comment:
Should we assert that the time taken for this RPC to complete is greater than the election timeout?

Reply:
It's a little tricky, we don't have a good measure, but we can look at client elapsed time, sure.